### DISCUSSION TITLE
Release Version 0.16.2: The Linda Lindas

### DISCUSSION THREAD

**Discussion by sgoggins** [chaoss/augur#1309] [General]
Updates In This Release Include: 
1. Dependency worker updates, 
2. Documentation Improvements 
3. MAJOR ease of use improvements to Docker implementation 
4. Updates to software dependencies in Python. 

<hr /><em>This discussion was created from the release <a href='https://github.com/chaoss/augur/releases/tag/0.16.2'>Release Version 0.16.2: The Linda Lindas</a>.</em>

**Comment by caifand** [comment]
Like the version name!
